frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -253,45 +308,69 @@ export const OperandTableRow: React.FC<OperandTableRowProps> = ({ obj }) => {
 
 export const OperandList: React.FC<OperandListProps> = (props) => {
   const { t } = useTranslation();
-  const Header = () => {
-    return [
-      {
-        title: t('public~Name'),
-        sortField: 'metadata.name',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[0] },
-      },
-      {
-        title: t('public~Kind'),
-        sortField: 'kind',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[1] },
-      },
-      {
-        title: t('public~Status'),
-        sortFunc: 'operandStatus',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[2] },
-      },
-      {
-        title: t('public~Labels'),
-        sortField: 'metadata.labels',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[3] },
-      },
-      {
-        title: t('public~Last updated'),
-        sortField: 'metadata.creationTimestamp',
-        transforms: [sortable],
-        props: { className: tableColumnClasses[4] },
-      },
-      {
-        title: '',
-        props: { className: tableColumnClasses[5] },
-      },
-    ];
+  const { toggleView, olmObj } = props;
+
+  const nameHeader: Header = {
+    title: t('public~Name'),
+    sortField: 'metadata.name',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  };
+  const kindHeader: Header = {
+    title: t('public~Kind'),
+    sortField: 'kind',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  };
+  const namespaceHeader: Header = {
+    title: t('olm~Namespace'),
+    sortFunc: 'metadata.namespace',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[2] },
+  };
+  const statusHeader: Header = {
+    title: t('public~Status'),
+    sortFunc: 'operandStatus',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[3] },
+  };
+  const labelsHeader: Header = {
+    title: t('public~Labels'),
+    sortField: 'metadata.labels',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[4] },
   };
 
+  const lastUpdatedHeader: Header = {
+    title: t('public~Last updated'),
+    sortField: 'metadata.creationTimestamp',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[5] },
+  };
+  const kebbaHeader: Header = {
+    title: '',
+    props: { className: tableColumnClasses[6] },
+  };
+
+  const AllNsHeader = (): Header[] => [
+    nameHeader,
+    kindHeader,
+    namespaceHeader,
+    statusHeader,
+    labelsHeader,
+    lastUpdatedHeader,
+    kebbaHeader,
+  ];
+  const CurrentNsHeader = (): Header[] => [
+    nameHeader,
+    kindHeader,
+    statusHeader,
+    labelsHeader,
+    lastUpdatedHeader,
+    kebbaHeader,
+  ];
+  const managesOtherNamespaces = hasOtherManagedNamespaces(props.olmObj);
```

Comment:
nit, this is already destructured
```suggestion
  const managesOtherNamespaces = hasOtherManagedNamespaces(olmObj);
```